plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -356,7 +356,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 			Rules: []rbacv1.PolicyRule{
 				// Used to build serviceLister
 				rbacv1helpers.NewRule("list", "watch").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
-				rbacv1helpers.NewRule("get").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
```

Comment:
I've tested, this works for list-watch node's metadata

Reply:
I might be missing context. We made an update to `nodeProxierRules` below, why the new role here?